src/lucky/web_socket_action.cr

```diff
@@ -0,0 +1,20 @@
+abstract class Lucky::WebSocketAction < Lucky::Action
+  @socket : HTTP::WebSocket?
+  @handler : HTTP::WebSocketHandler
+
+  def initialize(@context : HTTP::Server::Context, @route_params : Hash(String, String))
+    @handler = HTTP::WebSocketHandler.new do |ws|
+      @socket = ws
+      ws.on_ping { ws.pong("PONG") }
+      call
+    end
```

Comment:
This would create a new handler for every websocket action you added. I'm not sure how many a person would add, but it seems like you should only have 1, and the handler would just route to each action based on where it connected. This looks like what [Kemal](https://github.com/kemalcr/kemal/blob/master/src/kemal/websocket_handler.cr) seems to be doing too. A single instance that just adds each route and the handler.